core/src/main/scala/com/banno/kafka/IncomingRecord.scala

```diff
@@ -110,7 +110,7 @@ object IncomingRecord {
       metadata: Metadata,
   ) extends IncomingRecord[K, V]
 
-  // Exposing this as the only constructor allows us to add more fields that are
+  // Exposing this constructor allows us to add more fields that are
```

Comment:
This comment loses its meaning, or at least it needs to be reworded, with "as the only" dropped